ai-platform-unified/tutorials/unofficial/pytorch-image-classification-custom-train-container-gpu-custom-prediction-container-torchserve/model_server/convert_b64.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
```

Comment:
Nit: I think this file is only used in the "TorchServe Local Run" section of your README. Some organization suggestions:

1. Maybe it could be moved out of the `model_server` directory to the higher level directory? Because I believe everything else in the `model_server` directory is actually used in the prediction Docker container, so it's a little easier of a mental model for me skimming through here to think "`model_server` contains just the files used for the prediction container")

2. Since you have similar base64-related code in `sdk_online_predict.py`, maybe you can reuse this? E.g. Change the code a little and have `sdk_online_predict.py` import this one. (I'm actually not positive this would be much of an improvement, since it's kind of nice to have `sdk_online_predict.py` self-contained.

Alternatively, since you are using this script in the context of the command-line anyway, you could consider deleting this file and just use the `base64` UNIX command-line tool instead: Instead of `python convert_b64.py` I think you can do

```
echo "{
  \"instances\":  [{
    \"data\": {
      {\"b64\": \"$(base64 sample.jpg)\"}
    }
  }]
}" > sample_b64.json
```

(I have not tested this carefully.)

Just a couple of ideas; it also works fine the current way. 😄